README.md

```diff
@@ -3183,6 +3183,11 @@ Other Style Guides
       inventorOf,
     );
 
+    //good
+    export const MY_EXPORTED_FUNCTION = () => {
+  return 'hello function';
+};
```

Comment:
the indentation on this is incorrect, and there's no reason for this to be an arrow function, so it shouldn't be:
```suggestion
    export function MY_EXPORTED_FUNCTION() {
      return 'hello function';
    };
```

but also this example has nothing to do with dangling commas, which is the section you've placed it in

Reply:
this still doesn't make any sense in this section.